wallet/tutorials/javascript-dapp-simple.md

```diff
@@ -0,0 +1,372 @@
+For deployment in a production environment, we recommend [connecting to MetaMask using EIP-6963](../how-to/connect/index.md) instead. 
+
+[EIP-6963](https://eips.ethereum.org/EIPS/eip-6963) introduces an alternative wallet detection mechanism to the `window.ethereum` provider, and enables dapps to support [wallet interoperability](../concepts/wallet-interoperability.md).
+
+For a full end-to-end tutorial that can be used in production, see the
+[Create a simple React dapp](../tutorials/react-dapp-local-state.md) tutorial.
+:::
+
+## Prerequisites
+
+- [MetaMask](https://metamask.io/) [installed in the browser of your choice](/snaps/get-started/install-flask/) on your development machine.
+ We recommend [running a development network](/wallet/how-to/run-devnet/) on MetaMask when developing a dapp.
+
+- A text editor of your choice, such as [VS Code](https://code.visualstudio.com/).
+ You can install the [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer)
+extension for VS Code to easily launch a local development server for your dapp.
+
+- A module bundler, such as [Webpack](https://github.com/webpack/webpack).
+
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+
+## Steps
+
+### 1. Set up a new project
+
+Your dapp project directory must have the following structure:
+
+```text
+simple-dapp/
+├─ src/
+│  ├─ index.js
+├─ dist/
+│  ├─ index.html
+```
+
+For the dapp to work, your project script `index.js` must:
+
+- [Detect the MetaMask provider.](#2-detect-metamask)
+- [Determine a user's connected network.](#3-determine-a-users-connected-network)
+- [Access an account.](#4-access-an-account)
+
+#### Create a project
+
+Create and navigate into the `simple-dapp` directory using the following commands:
+
+```bash
+mkdir simple-dapp
+cd simple-dapp
+```
+
+:::caution important
+If you import any modules into your project, such as
+[`@metamask/detect-provider`](https://github.com/MetaMask/detect-provider), use a bundler such as
+[Webpack](https://github.com/webpack/webpack) to compile the modules and create an output script
+`dist/main.js`.
+See [Webpack's Getting Started guide](https://webpack.js.org/guides/getting-started/) for more information.
+:::
+
+In the `simple-dapp` directory, create `src` for the source files and `dist` for the distribution files that output from your Webpack builds. 
+Create a JavaScript file in `src`, then create an HTML file in `dist` from the `simple-dapp` root.
+Include the following HTML code example to establish a link to the JavaScript file generated by Webpack:
+
+```jsx title="index.html"
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Simple dapp</title>
+</head>
+<body>
+    <script src="../dist/index.js"></script>  <!-- Links to the JavaScript bundle created by Webpack -->
+</body>
+</html>
+```
```

Comment:
````suggestion
```html title="index.html"
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Simple dapp</title>
</head>
<body>
  <script src="../dist/index.js"></script>  <!-- Links to the JavaScript bundle created by Webpack -->
</body>
</html>
```
````